manifests/o/OKEx/OKExSetup/1.8.25/OKEx.OKExSetup.installer.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: OKEx.OKExSetup

```

```suggestion
PackageIdentifier: OKEx.OKEx
```
No reason for OKExSetup